config/ember-try.js

```diff
@@ -80,6 +80,14 @@ module.exports = async function () {
       },
       embroiderSafe(),
       embroiderOptimized(),
+      {
+        name: 'no-deprecations',
```

Comment:
so this should be added to the github action workflow (probably in the allowed to fail section 😂) so we actually run it with CI

Also I tend to also add an ember-release-no-deprecations one too that lets us know if we're getting new deprecations that we would need to deal with https://github.com/mansona/ember-cli-notifications/blob/main/test-app/config/ember-try.js#L133-L141 but that's probably getting ahead of ourselves 🙈

Reply:
Have added 👍